webapp_v2/src/components/common/TabWrapper.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
this shouldn't be in common since this is heavily dependent on study. It should go under components/singlestudy